internal/pkg/service/buffer/condition/checker.go

```diff
@@ -10,7 +10,6 @@ import (
 
 	"github.com/keboola/keboola-as-code/internal/pkg/log"
 	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/config"
-	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/file"
 	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/sink/tablesink/statistics"
```

Comment:
I must removed import of these removed packages, so `go mod tidy` and other commands works.

There are some broken files and tests, but refactored part of packages works, and we can continue:
![image](https://github.com/keboola/keboola-as-code/assets/19371734/a1c4bb3b-9936-4cb9-8fd0-cf0bf3aaa6b5)